packages/expo-updates/package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@expo/metro-config": "^0.1.70",
     "@expo/config-plugins": "^3.0.0",
+    "@expo/config": "5.0.6",
```

Comment:
all three of these packages depend on each other in various ways, be sure that they're all on latest